tests/sentry/relay/config/test_metric_extraction.py

```diff
@@ -122,14 +122,17 @@ def test_get_metric_extraction_config_single_alert(default_project: Project) ->
         config = get_metric_extraction_config(default_project)
 
         assert config
-        assert len(config["metrics"]) == 1
-        assert config["metrics"][0] == {
```

Comment:
I'm going to remove the usage of asserting specific indices of `config["metrics"]`.